README.md

```diff
@@ -1,9 +1,10 @@
-Terraform Provider For Alibaba Cloud
-==================
+# Terraform Provider For Alibaba Cloud
+![Travis CI Build Status](https://travis-ci.org/terraform-providers/terraform-provider-alicloud.svg?branch=master)
```

Comment:
Currently, the repo has been transferred to organization `aliyun` and please update it. Thanks.